common/src/Model/EntityAttributes.h

```diff
@@ -62,6 +62,7 @@ namespace TrenchBroom {
             extern const std::string GroupClassname;
             extern const std::string GroupTypeLayer;
             extern const std::string GroupTypeGroup;
+            extern const std::string DefaultAttributeValue;
```

Comment:
I would just call it `DefaultValue`, since the namespace `AttributeValues` already gives enough context.